lib/rubycritic/generators/html/view_helpers.rb

```diff
@@ -24,7 +24,10 @@ def smell_location_path(location)
     end
 
     def code_index_path(root_directory, file_name)
-      file_path("#{File.expand_path(root_directory)}/#{file_name}")
+      root_directory_path = File.expand_path(root_directory)
+      file_path = "#{root_directory_path}/#{file_name}"
+      file_path = "#{root_directory_path}/overview.html" unless File.exist?(file_path)
+      file_path(file_path)
```

Comment:
This is weird. What about naming the local var `index_path` or just `path` so not confuse it with the method name `file_path`?

Reply:
@nunosilva800  oh yes my apologies. How did I not see how bad that naming was. I have used index_path.